### ISSUE TITLE
TFLite produce wrong results when add follows a leakyrelu

### ISSUE THREAD

**Issue by Co1lin** [tensorflow/tensorflow#57818]
### 1. System information

Reproduced on 2.11.0.dev20220914 and 2.8.2 .

#### Option A: Reference colab notebooks

https://colab.research.google.com/drive/1VfnLOX68h0rMrTA3Fqk4moIOTLOL-P22?usp=sharing

#### Option B: Paste your code here or provide a link to a custom end-to-end colab

```python
import tensorflow as tf

def get_tflite_callable(model, inp_dict):
    converter = tf.lite.TFLiteConverter.from_concrete_functions(
        funcs=[model.__call__.get_concrete_function(**inp_dict)],
        trackable_obj=model,
    )
    converter.target_spec.supported_ops = [
        tf.lite.OpsSet.TFLITE_BUILTINS, # enable TensorFlow Lite ops.
        tf.lite.OpsSet.SELECT_TF_OPS # enable TensorFlow ops.
    ]
    tflite_bytes = converter.convert()
    interpreter = tf.lite.Interpreter(model_content=tflite_bytes)
    runner = interpreter.get_signature_runner()
    return runner

class MyModule(tf.Module):
    def __init__(self):
        super().__init__()
        self.const = tf.constant([-10.43154963850975037], dtype=tf.float64)

    @tf.function
    def __call__(self, x):
        c = tf.raw_ops.LeakyRelu(
            features=self.const, alpha=0.1,
        )
        x = tf.add(c, x)
        return x


inp = {
    "x": tf.constant([3.12363398], dtype=tf.float64),
}
m = MyModule()

out = m(**inp)
print(f'{out}') # t = <tf.Tensor: shape=(1,), dtype=float64, numpy=array([2.080479])>

runner = get_tflite_callable(m, inp)
out = runner(**inp)['output_0']
print(f'{out}\n{out.dtype}') # out = array([3.12363398])  out.dtype = dtype('float64')

```

### 3. Failure after conversion

Model produces wrong results and/or has lesser accuracy.

Actually, in this case, TFLite just outputs the input. It seems that the const number after leakyrelu and then added to the input is omitted.

**Comment by mohantym** [comment]
@Co1lin !
Thanks for reporting this bug.

@sachinprasadhs !
Could you look at this issue. Attached gist in [2.9](https://colab.sandbox.google.com/gist/mohantym/b94a849bff3ccc9bfdb03f47f95835b4/tflite_max_broadcasting.ipynb#scrollTo=gglvk92Ah3iq), [2.10](https://colab.sandbox.google.com/gist/mohantym/eb886f5184d6f691f9edb103f6a4a74b/tflite_max_broadcasting.ipynb#scrollTo=E3Gb0_08h6WU) and [nightly](https://colab.sandbox.google.com/gist/mohantym/377cb48c3fb47f0815f8592058b0ef5c/tflite_max_broadcasting.ipynb#scrollTo=ln0ycfR-igkI) for reference.

Thank you!

**Comment by sachinprasadhs** [comment]
I can confirm the behavior is only happening with `tf.nn.leaky_relu` / `tf.raw_ops.LeakyRelu` and it works well with the `Relu`.
Thanks for reporting the bug, we will look into it.

**Comment by pjpratik** [comment]
Issue still persists on TF 2.11 and TF Nightly. Please find the gist [here](https://colab.sandbox.google.com/gist/pjpratik/0ed7625f7f730f813cb90910a701db17/57818.ipynb). Thank you!

**Comment by sachinprasadhs** [comment]
Hi, The ideal way to compare would be to define a sample Keras model with leaky relu and compare the results between Keras model and TFLite converted model. 
Could you please provide those comparison. Thanks!

**Comment by sachinprasadhs** [comment]
Could you please follow the python API from the formats mentioned below https://www.tensorflow.org/lite/models/convert/convert_models#python_api_,

**Comment by pkgoogle** [comment]
Hi,

Thank you for opening this issue. Since this issue has been open for a long time, the code/debug information for this issue may not be relevant with the current state of the code base.

The TFLite team is constantly improving the framework by fixing bugs and adding new features. We suggest you try the latest TensorFlow [version](https://pypi.org/project/tensorflow/) with the latest compatible hardware configuration which could potentially resolve the issue. If you are still facing the issue, please create a new GitHub issue with your latest findings, with all the debugging information which could help us investigate.

Please follow the [release notes](https://github.com/tensorflow/tensorflow/blob/master/RELEASE.md) to stay up to date with the latest developments which are happening in the TFLite space.
